File: app/src/main/java/com/example/user/SmartBartender/SmartBartender.java
package com.example.user.SmartBartender;

import android.app.Application;
import java.util.ArrayList;
import java.util.Arrays;

public class SmartBartender extends Application {
    //настройки

    ArrayList<Integer> motors = new ArrayList<>(Arrays.asList(0,0,0,0,0,0));
    int coefficient = 1500;
    int numberOfMotors = 6;
    public SmartBartender(){
    }

    public void setMotors(ArrayList<Integer> motors) {
        this.motors = motors;
    }

    public void setCoefficient(int coefficient) {
        this.coefficient = coefficient;
    }

    public void setNumberOfMotors(int numberOfMotors) {
        this.numberOfMotors = numberOfMotors;
    }

    public ArrayList<Integer> getMotors() {
        return motors;
    }

    public int getCoefficient() {
        return coefficient;
    }

    public int getNumberOfMotors() {
        return numberOfMotors;
    }

    public void deleteSettings(){
        for(int i=0; i<numberOfMotors;i++){
            motors.add(i,0);
        }
    }
    public void deleteItemFromSettings(int item){
        int index;
        for(int i=0;i<numberOfMotors;i++) {
            index = motors.indexOf(item);
            if (index != -1) motors.set(index,0);
        }
    }
}

File: app/src/main/java/com/example/user/bartender/SmartBartender.kt
package com.example.user.bartender

import android.app.Application
import android.bluetooth.BluetoothAdapter
import com.example.user.bartender.database.Ingredient

class SmartBartender : Application(){
    val btAdapter: BluetoothAdapter = BluetoothAdapter.getDefaultAdapter()

    val ingredient = Ingredient("Пусто", 1500)
    private var settings = ArrayList<Ingredient>()
    init {
        for(i in 0..5){
            settings.add(ingredient)
        }
    }

    fun setMotors(motors: ArrayList<Ingredient>) {
        this.settings = motors
    }

    fun getMotors(): ArrayList<Ingredient> {
        return settings
    }

    fun clearSettings() {
        settings.clear()
        for(i in 0..5){
            settings.add(ingredient)
        }
    }
}
